chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -60,8 +63,9 @@
 end
 
 # PXE config
-%w(x86_64).each do |arch|
-  # Make it easy to totally disable/enable ppc64le
+# ppc64le bootloader can parse pxelinux config files
+%w(x86_64 ppc64le).each do |arch|
```

Comment:
The values I'm iterating over are the architectures for which PXE boot is used or EFI boot is used; it doesn't mean that it's the architectures for which we support discovery, that's why we have the `next` at the beginning of the loop.